docs/monitoring.md

```diff
@@ -477,7 +477,7 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
       A list of all stages for a given application.
         <br><code>?status=[active|complete|pending|failed]</code> list only stages in the given state.
         <br><code>?details=true</code> lists all stages with the task data.
-        <br><code>?taskStatus=[RUNNING|SUCCESS|FAILED|KILLED|PENDING]</code> lists stages only those tasks with the specified task status. Query parameter taskStatus takes effect only when <code>details=true</code>.
+        <br><code>?taskStatus=[RUNNING|SUCCESS|FAILED|KILLED|PENDING]</code> lists stages only those tasks with the specified task status. Query parameter taskStatus takes effect only when <code>details=true</code>. This also supports multiple <code>taskStatus</code> such as <code>?details=true&taskStatus=SUCCESS&taskStatus=FAILED</code>.
```

Comment:
Nit: we can try to point out it's OR:

`This also supports multiple <code>taskStatus</code> such as <code>?details=true&taskStatus=SUCCESS&taskStatus=FAILED</code> which will return all tasks matching any of specified task status`

Reply:
done